app/services/processors/jira_project_defect_escape_rate_updater.rb

```diff
@@ -12,12 +14,15 @@ def call
         bug.deep_symbolize_keys!
         bug_fields = bug[:fields]
 
+        changelog = bug[:changelog]
+        histories = changelog[:histories] if changelog.present?
```

Comment:
JIRA returns a key named `histories`? Interesting, I would have thought it was named `stories`

Reply:
Haha in fact, is not a story but the log of any change in an issue (https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/#api-rest-api-3-issue-issueidorkey-changelog-list-post)